kitsune/wiki/templates/wiki/includes/document_macros.html

```diff
@@ -46,7 +46,7 @@ <h1 class="title">{{ document.title }}</h1>
     {% if not fallback_reason %}
       {{ document.html|safe }}
     {% elif fallback_reason == 'no_translation' %}
-      {% if not document.is_archived %}
+      {% if document.is_localizable and document.current_revision.is_ready_for_localization and not document.is_archived %}
```

Comment:
Actually, if you use `document.current_revision.is_ready_for_localization` this Notice will show only if the current revision is ready for localization.
The fact is if any revision of a English Document is ready for localization, the notice should show to localizer to localize them.